types/create-react-class/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for create-react-class 15.6
+// Project: https://facebook.github.io/react/
+// Definitions by: Asana <https://asana.com>, AssureSign <http://www.assuresign.com>, Microsoft <https://microsoft.com>
```

Comment:
If you wrote these definitions, why are you crediting all these others?

Reply:
Well, I simply moved the definitions out of `react` and into these new packages. I thought it was more appropriate to keep the "Definitions by" in sync with `react`.

I can put my name on these new packages if you think that's more appropriate.

Reply:
Yes, please use your GH username here, you did create the package after all. :+1: